PackageInfo.g

```diff
@@ -10,7 +10,7 @@ SetPackageInfo( rec(
 
 PackageName := "CategoryConstructor",
 Subtitle := "Construct categories out of given ones",
-Version := "2022.01-07",
+Version := "2022.02-08",
```

Comment:
For the future: The last commit in the PR should be the one having the new package version.